src/translate_ligo.coffee

```diff
@@ -254,6 +254,8 @@ reserved_hash[config.op_list] = true
 spec_id_trans_hash =
   "now"       : "abs(now - (\"1970-01-01T00:00:00Z\": timestamp))"
   "msg.sender": "sender"
+  "tx.origin" : "source"
+  "block.timestamp" : "abs(now - (\"1970-01-01T00:00:00Z\": timestamp))"
```

Comment:
Should we move `abs(now - (\"1970-01-01T00:00:00Z\": timestamp))` to separate variable and have "now" and "block.timestamp" point to it